### ISSUE TITLE
[Image Classification][Model Training] CrossViT training settings

### ISSUE THREAD

**Issue by xperzy** [BR-IDL/PaddleViT#30]
**Describe your feature request**
Check and modify the training settings for CrossViT models, add missing processing and training methods.

**Describe the reference code or paper**
- Paper link: https://arxiv.org/pdf/2103.14899.pdf
- Official Code: https://github.com/IBM/CrossViT

**Describe the possible solution**
- Paper:
   - "... based on DeiT, and apply their default hyper-parameter for training..."
   - Augmentation: rand augmentation (m9n2), mixup(0.8), cutmix(1.0), random erasing(0.25)
   - Droppath: 0.1
   - label smoothing: 0.1
   - Epochs: 300 (30 warm-ups)
   - 32 GPUs 
   - Batch size: 4096
   - cosine lr decay, linear warmup
   - init lr: 0.004
   - weight decay: 0.05
   - optimizer: AdamW
   - warmup start lr: 1e-6
- Code:
   - dropout: 0.0
   - droppath: 0.1
   - clip-grad: None
   - min lr: 1e-5
- Model weights init:
   -  linear layer: trunc_normal (.02)
   - layernorm layer: constant weight (1.0), bias (0.0)
   -  official code [here](https://github.com/IBM/CrossViT/blob/3f7ab77c5728b1c31a2d5cc6185ee9c7e754d952/models/crossvit.py#L263)

**Additional context**

**Comment by xperzy** [comment]
This has been fixed in
